packages/ui/src/Modal.tsx

```diff
@@ -24,34 +23,12 @@ export const Modal: FC<ModalProps> = ({
   title
 }) => {
   return (
-    <Transition.Root as={Fragment} show={show}>
-      <Dialog
-        as="div"
-        className="fixed inset-0 z-10 overflow-y-auto"
-        onClose={() => onClose?.()}
-      >
-        <div className="flex min-h-screen items-center justify-center p-4 text-center sm:block sm:p-0">
-          <Transition.Child
-            as={Fragment}
-            enter="ease-out duration-100"
-            enterFrom="opacity-0"
-            enterTo="opacity-100"
-            leave="ease-in duration-100"
-            leaveFrom="opacity-100"
-            leaveTo="opacity-0"
-          >
-            <Dialog.Overlay className="fixed inset-0 bg-gray-500/75 transition-opacity dark:bg-gray-900/80" />
-          </Transition.Child>
-          <span className="hidden sm:inline-block sm:h-screen sm:align-middle" />
-          <Transition.Child
-            as={Fragment}
-            enter="ease-out duration-100"
-            enterFrom="opacity-0 translate-y-4 sm:translate-y-0 sm:scale-95"
-            enterTo="opacity-100 translate-y-0 sm:scale-100"
-            leave="ease-in duration-100"
-            leaveFrom="opacity-100 translate-y-0 sm:scale-100"
-            leaveTo="opacity-0 translate-y-4 sm:translate-y-0 sm:scale-95"
-          >
+    <Dialog.Root open={show}>
+      <Dialog.Portal>
+        <div className="flex min-h-screen items-center justify-center p-4 text-center sm:block sm:p-0 fixed inset-0 z-10 overflow-y-auto">
```

Comment:
is this div not suitable on any other of radix comps?

Reply:
No, it is not needed on other radix components like Menu.
Modal is a generic component called at multiple places.